gpu-operator/gpu-operator-kubevirt.rst

```diff
@@ -557,7 +562,7 @@ Open a terminal and clone the driver container image repository.
 
 .. code-block:: console
 
-   $ export PRIVATE_REGISTRY=my/private/registry VGPU_HOST_DRIVER_VERSION=580.82.07 OS_TAG=ubuntu22.04
+   $ export PRIVATE_REGISTRY=my/private/registry VGPU_HOST_DRIVER_VERSION=${recommended} OS_TAG=ubuntu22.04
```

Comment:
I don't think we use the ${recommended} placeholder value here. The VGPU host driver versions are versioned differently from vgpu guest driver and the vanilla data centre drivers

Reply:
OK, reverting back to 580.82.07